src/main/kotlin/no/nav/familie/ef/sak/repository/domain/Vedlegg.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.familie.ef.sak.repository.domain
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Column
+import org.springframework.data.relational.core.mapping.Table
+import java.time.LocalDateTime
+import java.util.*
+
+@Suppress("ArrayInDataClass")
```

Comment:
Legg til 
    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (javaClass != other?.javaClass) return false

        other as Vedlegg

        if (id != other.id) return false
        if (sakId != other.sakId) return false
        if (opprettetTid != other.opprettetTid) return false
        if (!data.contentEquals(other.data)) return false
        if (navn != other.navn) return false

        return true
    }

    override fun hashCode(): Int {
        var result = id.hashCode()
        result = 31 * result + sakId.hashCode()
        result = 31 * result + opprettetTid.hashCode()
        result = 31 * result + data.contentHashCode()
        result = 31 * result + navn.hashCode()
        return result
    }
i stedet. Vet ikke helt konsekvensene av å ha defekt hashCode og equals på jdbc-klasse.